test/integration/server.js

```diff
@@ -153,10 +153,12 @@ helpers.stubUtxos = function(server, wallet, amounts, cb) {
       return {
         txid: helpers.randomTXID(),
         vout: Math.floor(Math.random() * 10 + 1),
-        satoshis: helpers.toSatoshi(amount).toString(),
+        satoshis: helpers.toSatoshi(amount),
         scriptPubKey: address.getScriptPubKey(wallet.m).toBuffer().toString('hex'),
         address: address.address,
         confirmations: confirmations,
+        path: address.path,
```

Comment:
Both `path` and `publicKeys` attributes should not be part of the data returned by the stub as they are not included in the response from the block explorer. We need BWS to extract these from the list of addresses in the wallet. If you need these keys, please create a separate helper function.